x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```
@@ -901,14 +904,21 @@ const getDataStreams = async (
   }));
 };
 
-const rolloverDataStream = (dataStreamName: string, esClient: ElasticsearchClient) => {
+const rolloverDataStream = (
+  dataStreamName: string,
+  esClient: ElasticsearchClient,
+  logger: Logger
+) => {
   try {
     // Do no wrap rollovers in retryTransientEsErrors since it is not idempotent
     return esClient.indices.rollover({
       alias: dataStreamName,
     });
   } catch (error) {
-    throw new Error(`cannot rollover data stream [${dataStreamName}] due to error: ${error}`);
+    logger.warn(`Cannot rollover data stream [${dataStreamName}] due to error: ${error}`);
+    throw new PackageESError(
+      `Cannot rollover data stream [${dataStreamName}] due to error: ${error}`
+    );
```

:+1: this actually looks like a case where we should return a 500 error.